lib/IDE/CodeCompletion.cpp

```diff
@@ -1549,6 +1561,13 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
     Builder.addDeclDocCommentWords(llvm::makeArrayRef(Pairs));
   }
 
+  void setAssociatedDecl(const Decl *D, CodeCompletionResultBuilder &Builder) {
```

Comment:
I’m not sure I like the name `setAssociatedDecl()` here because it reads as though it’s a setter on `this`, when in fact it’s setting something in `Builder`. Could it either have a different name or a clarifying doc comment?

Reply:
I agree that this naming might be confusing. But in this class, there're many existing method that receive and modify `Builder` with similar naming scheme. If we want to use a different name, we'd rename other methods altogether in a separate PR. Doc-comment is a good idea though.

Reply:
Makes sense. I'll add a doc comment for now.